protocol/tags.go

```diff
@@ -60,7 +60,7 @@ const MsgOfInterestTagMaxSize = 45
 const MsgDigestSkipTagMaxSize = 69
 
 // NetPrioResponseTagMaxSize is the maximum size of a NetPrioResponseTag message
-const NetPrioResponseTagMaxSize = 838
+const NetPrioResponseTagMaxSize = 850
```

Comment:
I think we need to add a comment here to explain that: 
NetPrioResponseTagMaxSize stores the value of NetPrioResponseSignedMaxSize() in a constant to use from the protocol package.

Reply:
Line 51 above has a backref to `TestMaxSizesCorrect()` which has even more of an explanation